maptivist/src/main/webapp/script.js

```diff
@@ -12,9 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//MAPS API
 /** The global map variable is the initialized primary map for the webpage 
 and should be used for any map parameters or variables */
-var map;
+var map, created_markers;
```

Comment:
I'm starting to doubt the need for this array, because I feel like each time the map refreshes that array will have to be cleared. So it may just be more work to keep up with the array, but yes that's it's purpose.